docs/learn/learn-account-abstraction.md

```diff
@@ -0,0 +1,114 @@
+---
+id: learn-account-abstraction
+title: Polkadot's Account Abstraction
+sidebar_label: Account Abstraction
+description: Polkadot's Native Account Abstraction.
+keywords: [account, polkadot account, account abstraction, proxy, multisig, batch]
+slug: ../learn-account-abstraction
+---
+
+## Your Keys, Your Responsibility
+
+Account abstraction addresses the challenges of managing cryptographic keys representing accounts on
+blockchains. Accounts on blockchains represent entities, from an individual's identity to an
+institution. In Web3, you digitally sign any transaction or, more generally, any message using your
+private key. Data is recorded on a public ledger (usually blockchain-based) whose multiple copies of
+it are stored in computers participating in a P2P network.
+
+While the account’s private keys grant users control and ownership, losing them results in losing
+access to digital assets and fragmentation of your digital identity since you will need to create a
+new account with a new set of keys. This poses a hurdle for both users and developers regarding
+security and adoption.
+
+## Definition of Account Abstraction
+
+The term _abstraction_ refers to separating the user experience from the private key, enabling a
```

Comment:
I'm not sure if it's a good idea, but we could mention that the contept was derived from Ethereum first via erc4337